src/ResourceManager/Compute/AzureRM.Compute.Netcore.psd1

```diff
@@ -61,7 +61,8 @@ RequiredAssemblies = 'netcoreapp2.0\AutoMapper.dll',
     'netcoreapp2.0\Microsoft.Data.Edm.dll',
     'netcoreapp2.0\Microsoft.Data.OData.dll',
     'netcoreapp2.0\Microsoft.WindowsAzure.Storage.dll',
-    'netcoreapp2.0\System.Spatial.dll'
+    'netcoreapp2.0\System.Spatial.dll', 
+    '.\Microsoft.Azure.Commands.Common.Strategies.dll'
```

Comment:
>.\ [](start = 5, length = 2)

shouldn't this as well reference strategies.dll in the netcoreapp2.0 folder? btw there were some issues with .\ path on linux on one of the versions of .Net Core PS.

Reply:
@hovsepm I think, you are right. @twitchax please, confirm.